pkg/row/copier.go

```diff
@@ -90,16 +91,17 @@ func NewCopier(db *sql.DB, tbl, newTable *table.TableInfo, config *CopierConfig)
 		return nil, errors.New("dbConfig must be non-nil")
 	}
 	return &Copier{
-		db:            db,
-		table:         tbl,
-		newTable:      newTable,
-		concurrency:   config.Concurrency,
-		finalChecksum: config.FinalChecksum,
-		Throttler:     config.Throttler,
-		chunker:       chunker,
-		logger:        config.Logger,
-		metricsSink:   config.MetricsSink,
-		dbConfig:      config.DBConfig,
+		db:               db,
+		table:            tbl,
+		newTable:         newTable,
+		concurrency:      config.Concurrency,
+		finalChecksum:    config.FinalChecksum,
+		Throttler:        config.Throttler,
+		chunker:          chunker,
+		logger:           config.Logger,
+		metricsSink:      config.MetricsSink,
+		dbConfig:         config.DBConfig,
+		CopierEtaHistory: NewCopierEtaHistory(),
```

Comment:
This field and function can be made private, since they (at least currently) are not being called outside of this package.

Reply:
yeah, they kind of should be isolated from copier, but I wasn't sure where to put them (util maybe?)

Anyway, changed all the funcs to private